static/indicateNav.js

```diff
@@ -1,4 +1,14 @@
 document.addEventListener('DOMContentLoaded', () => {
+    // Get default section value
+    const defaultSectionValue = localStorage.getItem('default')
```

Comment:
@dexterousdhruv

Reply:
Or I can assign a fallback value incase the local storage is empty. What do you say ?
> const defaultSectionValue = localStorage.getItem('default') || null

Reply:
As you've already done the same for `section`, make use of that

Reply:
Hey @imxade ! Here's  what I could come up with....

I am removing the global `section` variable since we are not calling displayData by default, so it's redundant.

On absolute initial launch, the `defaultSectionValue` is actually empty, however once the section value is stored in localStorage, then it will show the stored section on every reload by default.

Plus, importing `section` from `index.js` by converting it to `module` was causing `displayData` to be `undefined`.


> On absolute initial launch defaultSectionValue will be `null`. Fix it with the following:
> 
> ```
>     const defaultSectionValue = section
> ```

Reply:
@dexterousdhruv 

let's proceed with my suggested approach for the time being. Implementing it as I've described will minimize the PR footprint and only address the issue in hand.

Given that some other functions rely on section, I recommend creating a separate PR later to incorporate your proposed improvements. When you make that PR, please include a comment detailing the modifications you've made and the rationale behind them.

Reply:
@imxade  As I said, importing `section` from `index.js` by converting it to `module` was causing `displayData` to be `undefined`. So doing as mentioned below isn't going to work
> 
> On absolute initial launch defaultSectionValue will be null. Fix it with the following:
>  ```
>      const defaultSectionValue = section
> ```
I suggest we proceed with this: 
 ```
  const defaultSectionValue = localStorage.getItem('default')
```
On absolute initial launch, the defaultSectionValue is actually empty, however once the section value is stored in localStorage, then it will show the stored section on every reload by default.

Reply:
it's working here and it doesn't have to be explicitly imported.

can you create a new branch and show me how you are doing it?